sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
> don't we also want to tie this to is_funded

Ideally - we shouldn't have to. No person with a valid CC should ever have a due more than 9.41. These look like more or less the same cases we found on https://github.com/gratipay/gratipay.com/pull/3980#issuecomment-206063806. Let's check back here once I've identified the issue there